Engage Release Notes & Version Specific Notes (#6791)
umbraco/UmbracoDocs: 13/umbraco-engage/upgrading/version-specific-upgrade-notes.md

{% endhint %}

## Breaking changes
#### v13.2.0
Introduced Razor Class Library support to serve static files for Engage, removing physical backoffice, views, and assets files from development projects.

> While this is not considered a breaking change, it is recommended to remove these folders from your project to avoid any potential conflicts in the future.

Reviewer: Attempted a rewrite of this one, to shorten the sentence a bit.
Let me know what you think!

```suggestion
While this is not considered a breaking change, it is recommended that such folders be removed from the project to avoid conflicts in the future.
```

Author: Looks good to me @sofietoft, thank you!! 😁
